tests/test_csc.py

```diff
@@ -367,8 +373,12 @@ def test_run(self):
             process = await asyncio.create_subprocess_exec(exe_name)
             try:
                 async with salobj.Domain() as domain:
-                    remote = salobj.Remote(domain=domain, name="ATPneumatics", index=0)
+                    remote = salobj.Remote(domain=domain, name="ATPneumatics")
                     await remote.start_task
+
+                    for i in range(5):
+                        await remote.evt_heartbeat.next(flush=True, timeout=LONG_TIMEOUT)
```

Comment:
You only need the LONG_TIMEOUT for the first one. Why so many? This will really slow down the test. Isn't 2 enough?

Reply:
You are right, we only need the timeout for the first one. I couldn't get the test to pass reliably when waiting for a single heartbeat (no idea why). Maybe two would be enough though.